packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -118,6 +118,10 @@ export class RouterliciousRestWrapper extends RestWrapper {
 		statusCode: number,
 		canRetry = true,
 	): Promise<IR11sResponse<T>> {
+		if (this.token === undefined) {
```

Comment:
It feels like it's equivalent to calling this.getToken();. no?
Maybe even better to make generateHeaders() async and let token fetch be closer to where it's needed?